test/con_cache_test.exs

```diff
@@ -1,80 +1,107 @@
 defmodule ConCacheTest do
   use ExUnit.Case, async: false
+  import ExUnit.CaptureLog
+
+  @opts [ttl: false]
```

Comment:
This mechanism, together with `with_opts` is a bit cryptic to read. I propose that instead of that, you add a private `start_cache` function along the lines of:

```elixir
defp start_cache(opts \\ []), do:
  ConCache.start_link(Keyword.merge([ttl: false], opts)
```

And then replace all occurrences of `ConCache.start_link` with `start_cache`.